app/lib/core/models/animal_transport_record.dart

```diff
@@ -31,6 +31,27 @@ class AnimalTransportRecord {
     @required this.identifier,
   });
 
+  factory AnimalTransportRecord.defaultAtr(
+          {ShipperInfo shipInfo,
```

Comment:
It might be excessive to have these optional parameters on a default ATR, because you could just use the regular constructor. But, I'm not mad about this either. Thanks.